controller/helper/process-trial.js

```diff
@@ -27,14 +28,40 @@ function processTrial (trial) {
         IRBID: trial.IRBID,
         start: startDate.format('L'),
         end: endDate.format('L'),
-        recruitedPercent: null,
-        unrecruitedPercent: null,
-        activePercent: null,
-        completedPercent: null,
+        recruitedPercent: allPercent.recruitedPercent,
+        unrecruitedPercent: allPercent.unrecruitedPercent,
+        activePercent: allPercent.activePercent,
+        completedPercent: allPercent.completedPercent,
         compliantPercent: null,
         noncompliantPercent: null,
         noncompliantCount: null
     };
 }
 
+/**
+ * Takes in a Trial model and processes to provide percent calculation
+ * @param {Trial} trial - a single Trial object
+ * @returns {Object} calculated Percent
+ */
+function processPercent (trial) {
+    const percent = 100;
+    const zeroPercent = 0;
+
+    if (trial.recruitedCount > zeroPercent) {
```

Comment:
@poojaRal I think the condition is handled in code. With recruited count = 0, code flow will behave as it is supposed to be. 
Means % Recruited = 0, % Unrecruited = 100, % Active  = 0 and % Completed = 0.
Screen shot for reference in case of trial.recruitedCount = 0
![screen shot 2016-03-06 at 1 27 16 pm](https://cloud.githubusercontent.com/assets/5720497/13556767/2ff9595e-e39f-11e5-8d60-ea92f7086ab8.png)